components/dashboard/src/components/PillLabel.tsx

```diff
@@ -4,12 +4,14 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
-export type PillType = "info" | "warn" | "success";
+export type PillType = "info" | "warn" | "success" | "warnStrong" | "subtle";
 
 const PillClsMap: Record<PillType, string> = {
     info: "bg-blue-50 text-blue-500 dark:bg-blue-500 dark:text-blue-100",
     warn: "bg-orange-100 text-orange-700 dark:bg-orange-600 dark:text-orange-100",
     success: "bg-green-100 text-green-700 dark:bg-green-600 dark:text-green-100",
+    warnStrong: "bg-orange-500 text-white dark:bg-orange-600",
+    subtle: "bg-gray-300 text-gray-800 dark:bg-gray-600 dark:text-gray-100",
```

Comment:
**suggestion**: Could we name this `neutral`? Thoughts?

Reply:
Makes more sense than `subtle` IMO. changed it